docs/components/input-field/demo/base-demo.md

```diff
@@ -19,8 +26,9 @@ export default class extends Component {
   @tracked errorMessage;
 
   @action
-  updateValue(value, e) {
+  handleChange(value, event) {
     this.value = value;
+    console.log({ value, event });
```

Comment:
Is this intentionally here, or forgot to remove?

Reply:
It was intentional so that folks can see what they get for `value` and `event`, but happy to remove.

Reply:
No, that's fine then, I just was not sure.